HousingFinanceInterimApi/Startup.cs

```diff
@@ -44,6 +43,9 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            // Load environment variables from .env file in parent directory if it exists
+            DotEnv.Fluent().WithEnvFiles(Path.Combine(Directory.GetCurrentDirectory(), "../.env")).Load();
```

Comment:
Maybe add 'if environment = local'?

Thinking it would be good to avoid having this run on prod.

Reply:
This would only affect the environment if it finds a .env file, if it does not it will do nothing

Reply:
I was thinking from the perspective of - it's an additional external dependency code execution - so higher attack surface when it comes to security.

Reply:
Fair point. With this it won't try to find a `.env` file if it's deployed
https://github.com/LBHackney-IT/housing-finance-interim-api/pull/136/commits/7a1c66266688abfa21bcf3c004faa86c90b929c4

Reply:
Better solution: https://github.com/LBHackney-IT/housing-finance-interim-api/pull/136/commits/b34f6ac0bb040bba4cf55be68358437fac18ad83